themes/root/templates/Email/form.phtml

```diff
@@ -1,13 +1,12 @@
 <?php foreach($this->fields as $data): ?>
-<?php $label = $data['label'] ?? null ?>
-<?php $isTextarea = $data['type'] === 'textarea'; ?>
-<?php if ($isTextarea): ?>
+<?php $label = $this->translate($data['label'] ?? '') ?>
+<?php if ($data['type'] === 'textarea'): ?>
 -----
 <?=$label ? "$label: " : ''?>
 <?=$data['value']?>
 
 -----
 <?php else: ?>
-<?= ($label ? "$label: " : '') . $data['value'] . PHP_EOL ?>
+<?= ($label ? "$label: " : '') . implode(', ', array_map([$this, 'translate'], (array)$data['value'])) . PHP_EOL ?>
```

Comment:
Is there ever a value that you DON'T want to translate? Could this cause weird things to happen to user input, or is the idea that we're always going to use textareas for free-form user input?

Reply:
Good point. Indeed there is, and this didn't make sense at all. I changed it to translate only values of checkbox, radio and select.